frame-metadata/src/v15.rs

```diff
@@ -50,6 +50,12 @@ pub struct RuntimeMetadataV15 {
 	pub ty: <PortableForm as Form>::Type,
 	/// Metadata of the Runtime API.
 	pub apis: Vec<RuntimeApiMetadata<PortableForm>>,
+	/// The type of the outer `RuntimeCall` enum.
+	pub call_enum_ty: <PortableForm as Form>::Type,
```

Comment:
By doing this, we now have duplicated data in the metadata... I'm for this change, but then we should remove the duplicated stuff, like `pallet.error`, `pallet.calls` etc.

Reply:
Technically we change the encoded format, so it will always require a new metadata version. However, we are in the process of releasing the next metadata version any way.

Reply:
That makes, sense. I'll create another PR to remove the duplicate data (ie `pallet. calls`) for the V15, thanks!

Reply:
Can go into the post of @jsdw where he announced the V15 freeze.